commit 397f7b83e6f98313f1a7e9550325202763506f1d
Author: Andy Lutomirski <luto@kernel.org>
Date:   Mon Aug 7 19:43:13 2017 -0700

    x86/asm/64: Clear AC on NMI entries
    
    commit e93c17301ac55321fc18e0f8316e924e58a83c8c upstream.
    
    This closes a hole in our SMAP implementation.
    
    This patch comes from grsecurity. Good catch!
    
    Signed-off-by: Andy Lutomirski <luto@kernel.org>
    Cc: Borislav Petkov <bp@alien8.de>
    Cc: Brian Gerst <brgerst@gmail.com>
    Cc: Denys Vlasenko <dvlasenk@redhat.com>
    Cc: H. Peter Anvin <hpa@zytor.com>
    Cc: Josh Poimboeuf <jpoimboe@redhat.com>
    Cc: Linus Torvalds <torvalds@linux-foundation.org>
    Cc: Peter Zijlstra <peterz@infradead.org>
    Cc: Thomas Gleixner <tglx@linutronix.de>
    Link: http://lkml.kernel.org/r/314cc9f294e8f14ed85485727556ad4f15bb1659.1502159503.git.luto@kernel.org
    Signed-off-by: Ingo Molnar <mingo@kernel.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linux-4.9.47/arch/x86/entry/entry_64.S
===================================================================
--- linux-4.9.47.orig/arch/x86/entry/entry_64.S
+++ linux-4.9.47/arch/x86/entry/entry_64.S
@@ -1231,6 +1231,8 @@ ENTRY(nmi)
 
 	ASM_CLAC
 
+	ASM_CLAC
+
 	/* Use %rdx as our temp variable throughout */
 	pushq	%rdx
 
Index: linux-4.9.47/dummy/rpi_1650_397f7b83e6f98313f1a7e9550325202763506f1d.txt
===================================================================
--- /dev/null
+++ linux-4.9.47/dummy/rpi_1650_397f7b83e6f98313f1a7e9550325202763506f1d.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
